spec/system/qualifications/user_submits_a_request_to_change_date_of_birth_spec.rb

```diff
@@ -102,5 +107,15 @@ def then_my_request_is_submitted
 
   def and_my_evidence_is_uploaded
     expect(DateOfBirthChange.last.evidence.attached?).to be true
+    expect(DateOfBirthChange.last.malware_scan_result).to eq "pending"
+  end
+
+  def when_the_pending_scan_result_is_fetched_after_a_delay
+    time = Time.zone.now + 1.minute
+    travel_to(time) { perform_enqueued_jobs(only: FetchMalwareScanResultJob) }
```

Comment:
Nit: You can use the `.from_now` method to achieve the same result.

```suggestion
    travel_to(1.minute.from_now) { perform_enqueued_jobs(only: FetchMalwareScanResultJob) }
```